pkg/scripts_test/check.go

```diff
@@ -111,6 +111,14 @@ func checkEnvTokenInConfig(c check) {
 	require.Equal(c.test, token, conf.Extensions.Sumologic.InstallToken, "install token is different than expected")
 }
 
+func checkHostmetricsConfigCreated(c check) {
+	require.FileExists(c.test, hostmetricsConfigPath, "hostmetrics configuration has not been created properly")
```

Comment:
Can you also check permissions and ownership here? We've had a bunch of issues related to this, and I want to ensure correctness for any new additions.